static/js/ducks/taxonomies.js

```diff
@@ -0,0 +1,25 @@
+import * as API from '../API';
+import store from '../store';
+
+export const FETCH_TAXONOMIES = 'skyportal/FETCH_TAXONOMIES';
+export const FETCH_TAXONOMIES_OK = 'skyportal/FETCH_TAXONOMIES_OK';
+
+export const fetchTaxonomies = () => (
+  API.GET('/api/taxonomy', FETCH_TAXONOMIES)
+);
+
+const reducer = (state={ taxonomyList: [] }, action) => {
+  switch (action.type) {
+    case FETCH_TAXONOMIES_OK: {
+      const taxonomies = action.data;
```

Comment:
if we do `const taxonomyList = ...` here, then line 17 below can simply be `taxonomyList`